<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFIND</class>
 <widget class="QDialog" name="CFIND">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>408</width>
    <height>334</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>408</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>281</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="QchkFind">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="QlineFind">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>271</width>
          <height>191</height>
         </size>
        </property>
        <property name="title">
         <string>记录时间</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>224</width>
           <height>141</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="QrbtnTime">
              <property name="text">
               <string>时间</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>98</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>开始时间：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="QdateSTime">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>结束时间：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="QdateETime">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <zorder>QdateETime</zorder>
        <zorder>label</zorder>
        <zorder>QdateSTime</zorder>
        <zorder>QrbtnTime</zorder>
        <zorder>label_2</zorder>
        <zorder>QdateSTime</zorder>
        <zorder>label</zorder>
        <zorder>QdateETime</zorder>
        <zorder>QrbtnTime</zorder>
        <zorder>label_2</zorder>
        <zorder>horizontalSpacer</zorder>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="QpbtnOk">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>98</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="QpbtnClose">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
